tests/components/spreadsheet.test.ts

```diff
@@ -256,6 +256,16 @@ test("Spreadsheet detects frozen panes that exceed the limit size at start", asy
   expect(notifyUser).toHaveBeenCalled();
 });
 
+test("Warns user when viewport is too small for frozen panes but stops warning after resizing/Unmounted", async () => {
+  setDefaultSheetViewSize(0);
+  const notifyUser = jest.fn();
+  const model = new Model({ sheets: [{ panes: { xSplit: 0, ySplit: 20 } }] });
+  ({ parent, fixture } = await mountSpreadsheet({ model }, { notifyUser }));
+  expect(notifyUser).toHaveBeenCalledTimes(0);
+
+  setDefaultSheetViewSize(1000); // reset the default view size
```

Comment:
It would be more robust to save the value before changing it to 0, instead of hard-coding 1000. That way if the value one day changes, this test won't risk setting back a wrong value.
You can use `getDefaultSheetViewSize()`